src/javaharness/java/arcs/api/ShellApi.java

```diff
@@ -1,11 +1,5 @@
 package arcs.api;
 
 public interface ShellApi {
-    void observeEntity(String entityJson);
-
-    String receiveEntity(String entityJson);
-
-    void chooseSuggestion(String suggestion);
-
-    void postMessage(String msgToSendToHost);
+    String receive(String json);
```

Comment:
Can we add brief interface doc comment, and comment on this method? I can never remember when reading this which side of the pipe this is. Who's receiving what? Is this Android Java receiving a message from the Shell, or the Shell receiving a message from Android Java?

Reply:
added comment here and in DeviceClient.java